Desktop/utilities/helpmodel.cpp

```diff
@@ -134,36 +133,36 @@ void HelpModel::showOrTogglePage(QString pagePath)
 
 void HelpModel::showOrToggleParticularPageForAnalysis(Analysis * analysis, QString helpPage)
 {
-	if(!analysis)
-	{
-		_analysis = nullptr;
-		
-		setVisible(false);
-		return;
-	}
-	
 	QString renderFunc, 
 			contentMD,
 			pagePath = convertPagePathToLower(helpPage == "" ? analysis->helpFile() : analysis->fullHelpPath(helpPage));
 	
-	
-	if(analysis == _analysis && pagePath == _pagePath && _visible)
+	if(!analysis || (analysis == _analysis && pagePath == _pagePath && _visible))
```

Comment:
You test here whether the analysis pointer is not null, but you use this analysis pointer in the line before...

Reply:
pagePath should be set in the else expression

Reply:
Oops

Reply:
I just wanted to remove the duplication...